pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/services/ReportPathGeneratorService.kt

```diff
@@ -41,7 +41,7 @@ class ReportPathGeneratorService(private val project: Project) {
      * @return the path to the report file
      */
     fun getReportMutationsFile(): Path {
-        var path = checkForDebugBuiltType()
+        val path = getReportPath()
```

Comment:
why remove that? It was introduced by @nikomall34 to handle specific android build configurations

Reply:
I dont really know why it has removed this, i think i migth have overwritten something there with old code while pushing